lib/vm/src/backends/cranelift.rs

```diff
@@ -2,7 +2,6 @@
 use wasmer_clif_backend::CraneliftCompiler;
 use wasmer_runtime_core::{
     backend::{Backend, Compiler},
-    codegen::{MiddlewareChain, StreamingCompiler},
```

Comment:
Odd this never complained...
I guess I must run `cargo clippy`  with `--features cranelift`